### ISSUE TITLE
Request new module: cp-catalogs-waiops

### ISSUE THREAD

**Issue by dimallya** [cloud-native-toolkit/software-everywhere#107]
**Related user story**
Cloud Pak for Watson AIOps #106 

**Type of module**
gitops

**If requesting a terraform module, what is the target cloud platform?**
ibm, aws, azure, etc

**What is the purpose of the module?**
Create Catalog Source required for CP4WAIOps

**What is the category of the module?**
software

**Comment by triceam** [comment]
@dimallya to be clear, this will only contain catalog sources needed for Watson AIOps?  Does this overlap with other catalog sources?  ex: https://github.com/cloud-native-toolkit/terraform-gitops-cp-catalogs

**Comment by dimallya** [comment]
As per KC documentation, though the operator catalog name is `ibm-operator-catalog`, image source (i.e. `icr.io/cpopen/ibm-operator-catalog:latest`) mentioned in the CP4WAIOps catalog source is different than that of one which is present in existing cp-catalog module's `ibm-operator-catalog` image source (i.e. `docker.io/ibmcom/ibm-operator-catalog`). Also, for event manager installation `noi-operator-catalog`  catalog source is required. Hence we requested for the new catalog module and planning to include these 2 catalog sources in the cp-catalog-waiops module.

However, we figured out that operators of Watson AIOps AI Manager, Event Manager and Infrastructure Automation - all are available within existing `ibm-operator-catalog` catalog-source present in ocp-latest-gitops cluster, with the latest version and same image-source as mentioned in KC. Hence we can consider not to have the new module. We will leverage catalog source created by the existing cp-catalog module. 

Hence closing the issue.
